components/atoms/ReportProblem.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import PropTypes from "prop-types";
+import DropDown from "./DropDown";
+import Button from "./Button";
+
+export default function ReportProblem(props) {
+  return (
+    <div className="pb-6 pt-6">
+      <DropDown text="Report a problem or mistake on this page">
+        <div className="pt-2">
+          <form action="" method="post">
+            <div className="container border border-gray-md rounded bg-gray-light-report pb-5">
+              <fieldset className="text-left pl-2">
+                <legend className="text-center pb-3">
+                  <span>
+                    <h3>
+                      <hr className="border-1 pb-3 border-gray-300" />
```

Comment:
Why the rule? Seems like we can drop this.

Reply:
Was basing it off of Canada.ca's design as they have a rule at the top. Can take it out if not necessary though
![Screenshot 2021-04-21 120749](https://user-images.githubusercontent.com/71025360/115586273-6cdcb900-a29a-11eb-964b-e8a0be698c2d.png)

Reply:
Yeah, I see it. It's not on the design system version, so I don't think we need it: https://design.canada.ca/common-design-patterns/report-problem.html 

Weird that they are using it on Canada.ca though.